src/index.js

```diff
@@ -1,92 +1,66 @@
-  listDiv.appendChild(listDivTitle);
-  const addTask = document.createElement('div');
-  addTask.className = 'add-task';
-  addTask.innerHTML = `
-  <input type="text" placeholder="Add a new task.." class="new-list-item add-item">
-  <img class="enter-key" src="${Enter}" alt="enter-key">
-  `;
-  listDiv.appendChild(addTask);
+const tasks = getStorage();
 
-  // Add list icon
-  const tickIcon = new Image();
-  tickIcon.src = Tick;
-  logoElement.appendChild(tickIcon);
-
-  // Add refresh icon
-  const refereshIconDiv = document.createElement('div');
-  refereshIconDiv.className = 'refresh-icon';
-  const refreshIcon = new Image();
-  refreshIcon.src = Refresh;
-  refereshIconDiv.appendChild(refreshIcon);
-  listDivTitle.appendChild(refereshIconDiv);
-
-  toDoList.forEach((list) => {
-    const listItem = document.createElement('div');
-    listItem.className = 'to-do-item';
-    listItem.innerHTML = `
-          <input id="done" type="checkbox" value="done">
-          <input type="text" value="${list.description}" class="new-list-item">
-          <img class="view-more" src="${ViewMore}" alt="view-more">
-    `;
-    listDiv.appendChild(listItem);
-  });
+// Delete an item from local storage
+const removeIndex = (index) => {
+  setStorage(Delete.deleteOne(getStorage(), index));
+  populateTasks(getStorage()); // eslint-disable-line
+};
 
-  const clearContent = document.createElement('div');
-  clearContent.className = 'clear-content';
-  clearContent.innerHTML = 'Clear all completed';
-  listDiv.appendChild(clearContent);
+const populateTasks = (arr) => {
+  currentTasks.innerHTML = '';
+  for (let i = 0; i <= arr.length; i += 1) {
+    // Add an item to local storage
+    const newDiv = document.createElement('div');
+    newDiv.className = 'to-do-item';
+    const tick = document.createElement('input');
+    const description = document.createElement('input');
+    description.className = 'task-description';
+    const menuImg = document.createElement('img');
+    menuImg.src = `${ViewMore}`;
+    tick.setAttribute('type', 'checkbox');
+    tick.id = `item${i}`;
+    newDiv.append(tick);
+    description.value = `${arr[i].description}`;
+    newDiv.append(description);
+    newDiv.append(menuImg);
+    currentTasks.appendChild(newDiv);
 
-  // Append elements to container
-  container.appendChild(logoElement);
-  container.appendChild(listDiv);
+    // Double click the input area to display the delete icon
+    description.addEventListener('dblclick', () => {
+      setStorage(arr);
+      newDiv.classList.add('edit-mode');
+      newDiv.innerHTML = `<input type="checkbox" id="${i}"></input><input id = "update${i}" class="update" type="text" value = "${arr[i].description}"></input><i id="delete${i}" class="fas fa-trash-alt"></i>`;
+      document.getElementById(`update${i}`).focus();
+      document.getElementById(`delete${i}`).addEventListener('click', () => {
+        removeIndex(i);
+      });
+    });
```

Comment:
- [ ] When I edited a task, it doesn't get updated in the local storage, and because it doesn't get updated in the local storage, when I reloaded the page the old task comes back. Also, I got an error on the console `undefined description` which I think is related to this issue of editing a task. Kindly fix the error and make the editing of a task possible by updating the local storage when a task is edited. See the console error below;

<img width="434" alt="chrome_JexLmfTS8z" src="https://user-images.githubusercontent.com/81468675/159883522-6bb9a361-ea7d-4080-be02-844270c2862d.png">

- [ ] Kindly find a way to exit the editing mode because after I tried editing a task, I couldn't exit the editing mode even after I clicked on other tasks, the yellow background color was still there. Kindly make it such that when a user is done editing a task, clicking on anywhere on the app or other tasks should exit the editing mode.

Reply:
Hi @julie-ify  thanks for the review. The edit functionality was not part of this milestone. We were only supposed to do the add and remove that's why the edit functionality is not currently working

Reply:
@Benmuiruri, Kindly know that the edit functionality is part of this milestone. I checked your project requirement and I saw it there `Implement a function for editing task descriptions.` It is also in the code review guideline to ensure that students implemented the edit functionality. Kindly check the [project requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md) again to be sure of this.

Reply:
You are right, I missed that. However, my 2 coding partners had their projects approved without the edit function. After today's morning session I submitted mine with just the add and remove as well.  Thanks for letting me know

Reply:
@Benmuiruri, You're welcome! Kindly implement it okay.

Reply:
Hi @julie-ify  I have added the edit feature. Thank you